sdk/rust/oak_runtime_compat/src/asylo/mutex.rs

```diff
@@ -0,0 +1,133 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// See https://github.com/rust-lang/rust/blob/6b561b4917e803c4be4ca44d8e552b680cb9e380/src/libstd/sys/unix/mutex.rs
```

Comment:
Without running an actual `diff`, the current file looks exactly the same as the one in this link; is this actually the case, or should a reader be aware of any differences? Either way, this comment should clarify it.

Reply:
Mutex is basically the same yeah, just import changed, does that need a comment?

Reply:
The mutex file only had a import change